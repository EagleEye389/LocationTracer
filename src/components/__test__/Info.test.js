import React from 'react';
import Info from '../Information/Information';
import {shallow} from 'enzyme';

let component;
let testData="test";
describe("Testing Info Control",()=>{

 beforeEach(() => {
        component = shallow(<Info label={testData} value={testData} classStyle={testData} />);
  });

 it('Snapshot test', () => {
    expect(component).toMatchSnapshot();
  });

  it('Label prop test',()=>{
    expect(component.props())

  });

})