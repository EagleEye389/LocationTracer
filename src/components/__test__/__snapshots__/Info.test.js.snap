// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Info Control Snapshot test 1`] = `ShallowWrapper {}`;
